src/monster.cpp

```diff
@@ -2053,3 +2053,34 @@ void monster::hear_sound( const tripoint &source, const int vol, const int dist
         wander_to( tripoint( 2 * posx() - target_x, 2 * posy() - target_y, 2 * posz() - source.z ), wander_turns );
     }
 }
+
+void monster::on_unload()
+{
+    last_updated = calendar::turn;
+}
+
+void monster::on_load()
+{
+    // Possible TODO: Integrate monster upgrade
+    const int dt = calendar::turn - last_updated;
+    last_updated = calendar::turn;
```

Comment:
I think you did the right thing here, but I just wanted to bring it up.
The only time dt will be less than 0 is if we debugged the calendar value (pretty much don't care about things being consistent there) or if the monster is loaded in a new game.  When interacting between saves at different times, we pretty much have two options, either monsters will advance too rapidly, or they won't advance at all sometimes.  This does the first, if you move ```last_updated = calendar::turn;``` under the if conditional, we'd get the second behavior instead.
I think what you've done is the right thing, because I expect the dominant use case to be "play one game for a while, die, start new game" rather than "toggle back and forth between two games".
Might want a comment here outlining that we're making that tradeoff.